rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -116,9 +108,19 @@ def ensure_global(name, default):
     print(f"[rosdoc2] adding markdown parser", file=sys.stderr)
     # The `myst_parser` is used specifically if there are markdown files
     # in the sphinx project
-    # TODO(aprotyas): Migrate files under the `include` tag in the project's Doxygen
-    # configuration into the Sphinx project tree used to run the Sphinx builder in.
     extensions.append('myst_parser')
+    # If markdown support is enabled, the appropriate Pygments lexer must
+    # be registered for `.md`/`.markdown` file suffixes
+    exhale_args.update({{
```

Comment:
Will this work if `exhale_args` is `False` but `support_markdown` is `True`?

Reply:
Good catch. This will not work if `enable_exhale` is `False`. This could be a way to check for that but maintain the logical structure.
```suggestion
    if rosdoc2_settings.get('enable_exhale'):
        exhale_args.update({{
```

Reply:
Addressed in 6281b49.

An alternative to that patch would be to invoke `ensure_global` on `exhale_args`, but that may create a needless copy of `exhale_args` incase `enable_exhale` is false.